member-service/src/test/java/com/example/memberservice/member/service/MemberInternalServiceTest.java

```diff
@@ -0,0 +1,154 @@
+package com.example.memberservice.member.service;
+
+
+import com.example.memberservice.common.model.vo.Provider;
+import com.example.memberservice.member.entity.Members;
+import com.example.memberservice.member.entity.vo.Gender;
+import com.example.memberservice.member.repository.MemberJpaRepository;
+import com.example.memberservice.member.service.model.dto.output.MemberExistOutput;
+import com.example.memberservice.member.service.model.dto.output.MemberInfoOutput;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.tuple;
+
+@SpringBootTest
+@ActiveProfiles("test")
+class MemberInternalServiceTest {
+
+    @Autowired
+    private MemberInternalService memberInternalService;
+
+    @Autowired
+    private MemberJpaRepository memberJpaRepository;
+
+    @AfterEach
+    void tearDown() {
+        memberJpaRepository.deleteAllInBatch();
+    }
+
+    @Test
+    @DisplayName("Member Code List를 파라미터로 받았을 때 해당 Code에 유저에 대한 NickName, code, canWork가 주어진다. ")
+    void getMemberInfosTest(){
+        //Given
+        String code1 = UUID.randomUUID().toString();
+
+        String code2 = UUID.randomUUID().toString();
+
+
+        List<Members> members = List.of(
+            createMember(code1, "AAA@gmail.com","FirstUser", true,false),
+            createMember(code2, "BBB@gmail.com","SecondUser", false,false)
+        );
+
+        memberJpaRepository.saveAll(members);
+
+        //When
+        MemberInfoOutput resultMemberInfos = memberInternalService.getMemberInfos(List.of(code1, code2));
+
+        System.out.println(resultMemberInfos);
+        //Then
+        assertThat(resultMemberInfos.memberInfos())
+            .hasSize(2)
```

Comment:
사소한 부분이지만, 저는 검증하려는 길이를 하드코딩하기보다는 리스트의 크기를 비교하는 편입니다.
 
```suggestion
            .hasSize(members.size())
```

Reply:
헉 이거 좋은 방법인 것 같습니다!